verifier/verifier.go

```diff
@@ -67,11 +68,14 @@ func (v *verifier) Verify(ctx context.Context, desc ocispec.Descriptor, signatur
 	artifactRef := opts.ArtifactReference
 	envelopeMediaType := opts.SignatureMediaType
 	pluginConfig := opts.PluginConfig
+	logger := log.GetLogger(ctx)
 
+	logger.Debugf("Verify signature against artifact %v referenced as %s", desc.Digest, artifactRef)
```

Comment:
Should we log `envelopeMediaType` here for verify?

Reply:
Updated.